src/App.vue

```diff
@@ -18,8 +26,23 @@ const traitValues = ref<number[]>([0]);
 const addTrait = () => traitValues.value.push(0);
 const removeTrait = (index: number) => (traitValues.value = traitValues.value.filter((_, idx) => idx !== index));
 
-const stats = ['Combat', 'Exploration', 'Industrial', 'Trade'];
-const statValues = ref<number[]>(Array.from({ length: stats.length }, () => 0));
+const stats = ref([
```

Comment:
If this was a `reactive`, then the `.value` in the `statValues` variable assignment wouldn't be necessary.